melodic/distribution.yaml

```diff
@@ -6027,6 +6027,12 @@ repositories:
       url: https://github.com/peci1/movie_publisher.git
       version: melodic-devel
     status: developed
+  moving_average:
```

Comment:
`moving_average` is a generic name, is there something more specific that it can be named?

Reply:
Any suggestion? For a package that computes moving averages I don't see what could be a better name.